src/HonzaBotner/appsettings.json

```diff
@@ -11,7 +11,8 @@
         "GuildId": 721264747466719285
     },
     "CommonCommandOptions": {
-        "MuteRoleId": 750276752663904306
+        "MuteRoleId": 750276752663904306,
+        "HugEmoteName": "peepoHugger"
```

Comment:
Rather use id? `772088050615582760`

Reply:
This should be fixed in #57 rather... and set on deployment env anyway.